relayer/msgs.go

```diff
@@ -320,3 +323,195 @@ func (c *Chain) MsgTransfer(dst *PathEnd, amount sdk.Coin, dstAddr string,
 		timeoutTimestamp,
 	)
 }
+
+// MsgRelayTimeout
```

Comment:
we need to add some go doc (we need to explain which chain is sender and which is receiver)